### PR TITLE
Issue /#261 leafletData directiveControls id extension fixed

### PR DESCRIPTION
**Pull request by nmccready**
fix(leafletData unresolveMap): argument naming renamed scopeId to mapId

### CODE REVIEW

**Comment by nmccready** [comment]
@tallented

File: test/unit/services/leafletHelpersSpec.coffee [hunk 1]
```diff
@@ -8,46 +8,57 @@ describe 'leafletHelpers', ->
...
-          object = { foo: { sea: 'hawks' }}
-          expect(@subject.getObjectValue(object, 'foo.sea')).toEqual('hawks')
-          expect(@subject.getObjectValue(object, 'foo.sea.birds')).toEqual(undefined)
-          expect(@subject.getObjectValue(object, 'boo.hoo')).toEqual(undefined)
+            object = { foo: { sea: 'hawks' }}
+            expect(@subject.getObjectValue(object, 'foo.sea')).toEqual('hawks')
+            expect(@subject.getObjectValue(object, 'foo.sea.birds')).toEqual(undefined)
+            expect(@subject.getObjectValue(object, 'boo.hoo')).toEqual(undefined)
+
+    describe 'obtainEffectiveMapId', ->
```
Comment (nmccready) [inline test/unit/services/leafletHelpersSpec.coffee]: Testing `obtainEffectiveMapId` better to make sure I know what is going on.

File: test/unit/eventsDirectiveSpec.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ describe('Directive: leaflet', function() {
           if(mapId)
             mapId = mapId + '.';
 
-          leafletData.getMap().then(function(map) {
+          leafletData.getMap(mapId).then(function(map) {
```
Comment (nmccready) [inline test/unit/eventsDirectiveSpec.js]: Main Important Fix!!!

File: test/unit/leafletData/leafletDataService.DirectiveControlsSpec.coffee [hunk 3]
```diff
@@ -48,41 +63,59 @@ describe 'leafletData directiveControls', ->
...
+                .then (controls) =>
                     expect(controls[@rootName].clean).toBeDefined()
 
-    describe 'geojson controls', ->
-        beforeEach ->
-            @rootName = 'geojson'
-        describe 'exists', ->
+
+        describe "#{directiveName} MANY controls", ->
+            beforeEach ->
```
Comment (nmccready) [inline test/unit/leafletData/leafletDataService.DirectiveControlsSpec.coffee]: These tests were valuable especially after `obtainEffectiveMapId` was changed to throw on multiple maps and no mapId provided. Once the [main fix](https://github.com/angular-ui/ui-leaflet/pull/262/files#r66184840) was put in these test passed.

**Comment by nmccready** [comment]
@elesdoar @jessertaylor @tallented  please review

**Comment by nmccready** [comment]
Once this is good to go another 1.X will be released.

**Comment by tallented** [comment]
LGTM

**Pull request merged**